troubleshooting.md

```diff
@@ -2,7 +2,7 @@
 
 Before you start to report a new issue on the GitHub repository, make sure to check your local installation, as some settings may typically result in an unexpected outcome of your WP-CLI commands.
 
-### How do I get more verbose informations about my WP-CLI installation?
+### How does I get more verbose informations about my WP-CLI installation?
```

Comment:
This is grammatically wrong, the original sentence was correct. Please revert this change.

Reply:
@schlessera Done